docs/versioned/spark/spark-queries.md

```diff
@@ -168,7 +168,7 @@ To inspect a table's history, snapshots, and other metadata, Iceberg supports me
 Metadata tables are identified by adding the metadata table name after the original table name. For example, history for `db.table` is read using `db.table.history`.
 
 {{< hint info >}}
-As of Spark 3.0, the format of the table name for inspection (`catalog.database.table.metadata`) doesn't work with Spark's default catalog (`spark_catalog`). If you've replaced the default catalog, you may want to use `DataFrameReader` API to inspect the table. 
+In Spark 3.0 and 3.1, the Spark session catalog (`spark_catalog`) does not support table names with multipart identifiers such as `catalog.database.table.metadata`. To work around this, for querying metadata tables, configure a different catalog that uses the Iceberg `SparkCatalog` class, or use the Spark `DataFrameReader` API. From Spark 3.2 onwards, the session catalog supports table names with multipart identifiers.
```

Comment:
Nit: Consider `Prior to Spark 3.2` instead of `Spark 3.0 and Spark 3.1`.

I don't think this applies at all to Spark 2.4, but I can still imagine getting questions about it.

Reply:
Spark 2.4 does not support querying tables using SQL, so this does not apply to 2.4 at all. That is why I wrote "In Spark 3.0 and 3.1", since querying using Spark SQL is only supported in Spark 3 onward.
One can argue both ways about "Prior to Spark 3.2": The reader knows that SQL is not supported for Spark 2, so implicitly "prior to Spark 3.2" means 3.0 and 3.1. Or "prior to Spark 3.2" implies that this section applies to 2.4, which would be misleading.

Reply:
@kbendick I updated the text.
I added a line about Spark 2.4, pointing the reader to the section on "Inspecting with DataFrames".
I also used "prior to 3.2".